conf/etc/v1/ha.conf

```diff
@@ -0,0 +1,5 @@
+LOG:
+    path: /var/log/seagate/cortx/ha
+    level: INFO
+VERSION:
```

Comment:
Don't add new thing to conf file in v1. While parsing if version is not present, initialize it to 1.0.

Reply:
@ajay-paratmandali  told me to add this, anyway removing it now.

Reply:
In case if we need to modify something for version 1 then we need to check from code if version is 1 in future.
Also if we keep modifying file and written
`Conf.get(version)`
then v1 build may get failed so suggested to keep in both ha.conf

Reply:
I think we can tackle this easily, If we do get the the version from conf then it will return nothing from conf file means its version 1, else its version 2.

Reply:
Ok. Let's keep it.